src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -186,9 +190,32 @@ private Rotation2d getGyroscopeRotation() {
             return Rotation2d.fromDegrees(m_navx.getFusedHeading());
         }
 
-        // We have to invert the angle of the NavX so that rotating the robot
-        // counter-clockwise makes the angle increase.
-        return Rotation2d.fromDegrees(360.0 - m_navx.getYaw());
+		// We have to invert the angle of the NavX so that rotating the robot
+		// counter-clockwise makes the angle increase.
+		return Rotation2d.fromDegrees(360.0 - m_navx.getYaw());
+	}
+	// know the robot heading and get pitch and roll
+    private Translation3d getNormalVector3d() {
+		Rotation3d tilt = new Rotation3d(Units.degreesToRadians(m_navx.getRoll()), Units.degreesToRadians(m_navx.getPitch()), 0);
```

Comment:
We have been using "Math.toRadians()" for these conversions. Better to use the Java standard library.